common/src/chain/transaction/input.rs

```diff
@@ -9,6 +9,8 @@ pub struct OutPoint {
 }
 
 impl OutPoint {
+    pub const COINBASE_OUTPOINT_INDEX: u32 = u32::MAX;
```

Comment:
I don't get why it's max... isn't it 0 in bitcoin?

Reply:
Nope. [Here](https://github.com/bitcoin/bitcoin/blob/8c0f02c69d50761c3a6d4dfb5d7cba43a00a55ae/src/primitives/transaction.h#L32)

Reply:
Yes, but this is not COINBASE index... this is a "null" initial value to know that there's no index associated with it. It's kind of a crappy None value. Why are we calling it coinbase?

Reply:
I'm resolving this because it sounds irrelevant currently.

Reply:
We no longer have coinbase, so I removed this.